server/handlers/clients.js

```diff
@@ -223,20 +222,36 @@ async function updateOrAdd(req, res) {
     else if (forceBool === 'false' || forceBool === false) forceBool = false;
```

Comment:
Looking at this logic around `forceBool` again, I think that if an object is sent as `force`, the object will get through, which is unintended behavior, maybe something like:

```
let forceBool = false;
if (force === 'true' || force === true) forceBool = true
```
Then the wrong type will be ignored and it will be left as false

Reply:
done 👍